docs/api-docs/storefront/graphql/graphql-api-overview.md

```diff
@@ -300,18 +285,15 @@ As a best practice, you should create tokens that expire and rotate them regular
 
 You have two options for public URLs you can use to run requests:
 
-
 - The store's default storefront URL, e.g. `https://store.com/graphql`
 - The store's permanent URL, e.g. `https://store-STOREHASH.mybigcommerce.com/graphql`
 
 #### I want to run requests from the perspective of another Channel
 
 You should use the Channel's Permanent URL of the form `https://store-STOREHASH-CHANNELID.mybigcommerce.com/graphql`.
 
-
 For example, if your store hash is `abc123` and your channel ID is `456`, the correct URL would be `https://store-abc123-456.mybigcommerce.com/graphql`.
 
-
 Note that you must create your Storefront API Token with the same channel ID, or your request will be rejected.

```

Comment:
```suggestion
You must create your storefront API token with the same channel ID, or the server will reject your request.
```

Reply:
converted to callout; improved by incorporating this suggestion.